conf/messages.cy

```diff
@@ -17,9 +17,13 @@ cf.account.details.row.DutyDeferment.info=Gohiriad tollau 1720
 cf.account.detail.download-link=Lawrlwytho {0} ar gyfer {1} i {2} ({3})
 cf.account.detail.supplementary-download-link=Lawrlwytho manylion diwedd mis atodol {0} ar gyfer {1} ({2})
 cf.account.detail.excise-download-link=Lawrlwytho crynodeb o’r ecséis {0} ar gyfer {1} ({2})
+cf.account.detail.download.excise-deferment-1920=Lawrlwythwch ohiriad ecséis 1920 ar ffurf {0} ar gyfer {1} ({2})
```

Comment:
I'm just double checking that as we're only adding four new text msgs but there are 8 in the story. I am assuming the others are all pre-existing?

Reply:
@AdamCoops  It’s actually 4. If you look closely, the other 4 are identical, differing only in their dynamic values.